src/openrct2/drawing/Drawing.h

```diff
@@ -296,6 +296,33 @@ enum class FilterPaletteID : int32_t
     PaletteGlassDarkPink = PaletteGlass + COLOUR_DARK_PINK,
```

Comment:
FilterPaletteId is meant to be a super set of colour_t. All of these offsets need to be moved down to make space for the new ones.